### PR TITLE
Database schema upgrade: Delete unused slug; test suite description column

### PR DESCRIPTION
**Pull request by okurz**
Needs database schema bump.

### CODE REVIEW

**Comment by okurz** [comment]
can someone explain why I get the error message

```
t/deploy.t .. 1/? Can't call method "id" on an undefined value at t/deploy.t line 86 (#1)
```

now?

**Comment by aaannz** [comment]
I think that the deploy test is not correct anymore. When upgrade is being tested, deploy test installs old schema version directly using deployment handler. This handler does not create system user as `_try_deploy_db` does. Then from that old version to your new one migration does not cover creation of system user, thus test fails.

**Comment by coolo** [comment]
Then let's take https://github.com/os-autoinst/openQA/pull/1095 and have try_deploy_db take a version to deploy

**Comment by coolo** [comment]
makes me wonder why the deploy.t isn't failing in #1094

File: lib/OpenQA/Schema/Result/JobTemplates.pm [hunk 1]
```diff
@@ -52,10 +52,10 @@ __PACKAGE__->belongs_to(test_suite => 'OpenQA::Schema::Result::TestSuites', 'tes
 __PACKAGE__->belongs_to(group      => 'OpenQA::Schema::Result::JobGroups',  'group_id');
 __PACKAGE__->add_unique_constraint([qw(product_id machine_id test_suite_id)]);
 
-sub variables {
+sub description {
     my $self = shift;
 
-    $self->machine->variables . $self->test_suite->variables . $self->product->variables;
+    $self->machine->description . $self->test_suite->description . $self->product->description;
```
Comment (coolo) [inline lib/OpenQA/Schema/Result/JobTemplates.pm]: I don't think this simple concat makes much sense.

**Comment by okurz** [comment]
Updated:
* rebased
* included #1095
* merged database schema

test fails now with `The command "openssl aes-256-cbc -K $encrypted_e2c381aa6b8c_key -iv $encrypted_e2c381aa6b8c_iv -in .openqa-travis.enc -out .openqa-travis -d" failed and exited with 1 during .` what is that?

**Comment by coolo** [comment]
please rebase

**Comment by okurz** [comment]
rebased, test still running, at least passed previous failure already

**Pull request merged**